main.js

```diff
@@ -0,0 +1,271 @@
+const myAllHTML = document;
+
+const products = [
+  {
+    productName: "bread",
+    productData: {
+      weight: 200,
+      certificate: true,
+      dateOfExpiry: "27/10/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry:
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/2560px-Flag_of_Ukraine.svg.png",
+    productPrice: 11.7,
+  },
+  {
+    productName: "milk",
+    productData: {
+      weight: 1000,
+      certificate: true,
+      dateOfExpiry: "28/10/2022",
+      sugarFree: true,
+    },
+    productProvider: "FOP Fil D.V.",
+    productCountry:
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/2560px-Flag_of_Ukraine.svg.png",
+    productPrice: 20.55,
+  },
+  {
+    productName: "sugar",
+    productData: {
+      weight: 900,
+      certificate: true,
+      dateOfExpiry: "28/10/2023",
+      sugarFree: false,
+    },
+    productProvider: "LLC Dolina-99",
+    productCountry:
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/2560px-Flag_of_Ukraine.svg.png",
+    productPrice: 22.0,
+  },
+  {
+    productName: "butter",
+    productData: {
+      weight: 200,
+      certificate: true,
+      dateOfExpiry: "28/12/2022",
+      sugarFree: false,
+    },
+    productProvider: "LLC Yagotynske",
+    productCountry:
+      "https://upload.wikimedia.org/wikipedia/commons/thumb/4/49/Flag_of_Ukraine.svg/2560px-Flag_of_Ukraine.svg.png",
+    productPrice: 60.7,
+  },
+  {
+    productName: "coffe",
+    productData: {
+      weight: 1000,
+      certificate: true,
+      dateOfExpiry: "28/12/2023",
+      sugarFree: true,
+    },
+    productProvider: "Varka GmbH",
+    productCountry:
+      "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAASIAAACuCAMAAAClZfCTAAAAElBMVEUAAAD/zgDdAADnAADaAAD/2AAtsSEoAAAA+ElEQVR4nO3QMQGAMAAEsYeCf8tIuI0pkZANAAAAAAAAAAAAAAAAAAAAgB8dwm6CoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUqKkqKkKClKewh7CbsIipKipCgpSoqSoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUqKkqKkKClKipKipCgpSoqSoqQoKUofMGTNC8HkSxoAAAAASUVORK5CYII=",

```

Comment:
Is is super not-efficient to keep images in a such way. Consider, at least, short url and then pass country be like:

`httts://www.imagesfrom.com/county/${productCountry}`

and `productCountry` will be `UA` as an example